x/emissions/module/reputer_rewards.go

```diff
@@ -0,0 +1 @@
+package module
```

Comment:
maybe at this point it would be appropriate to put all things "rewards" in its own folder